app/assets/stylesheets/autocomplete.scss

```diff
@@ -1,6 +1,12 @@
 .constituency-autocomplete {
     position: relative;
     display: inline-block;
+
+    // Fill the width to match the BS style
+    width: 100%;
```

Comment:
Quite a bit of wrangling required to get this looking right, will deserve some testing on different browsers